README.md

```diff
@@ -45,6 +45,41 @@ For a more complete introduction to using WP-CLI, read the [Quick Start guide](h
 
 Already feel comfortable with the basics? Jump into the [complete list of commands](https://developer.wordpress.org/cli/commands/) for detailed information on managing themes and plugins, importing and exporting data, performing database search-replace operations and more.
 
+### HTTP Request Logging
```

Comment:
We would not add this to the readme, but to the [handbook](https://github.com/wp-cli/handbook) instead